torchvision/csrc/models/squeezenet.cpp

```diff
@@ -0,0 +1,123 @@
+#include "squeezenet.h"
+
+#include <limits>
+#include "modelsimpl.h"
+
+namespace vision {
+namespace models {
+class Fire : public torch::nn::Module {
+  torch::nn::Conv2d squeeze, expand1x1, expand3x3;
+
+ public:
+  Fire(
+      int64_t inplanes,
+      int64_t squeeze_planes,
+      int64_t expand1x1_planes,
+      int64_t expand3x3_planes)
+      : squeeze(torch::nn::Conv2dOptions(inplanes, squeeze_planes, 1)),
+        expand1x1(
+            torch::nn::Conv2dOptions(squeeze_planes, expand1x1_planes, 1)),
+        expand3x3(torch::nn::Conv2dOptions(squeeze_planes, expand3x3_planes, 3)
+                      .padding(1)) {
+    register_module("squeeze", squeeze);
+    register_module("expand1x1", expand1x1);
+    register_module("expand3x3", expand3x3);
+  }
+
+  torch::Tensor forward(torch::Tensor x) {
+    x = torch::relu(squeeze->forward(x));
+    return torch::cat(
+        {torch::relu(expand1x1->forward(x)),
+         torch::relu(expand3x3->forward(x))},
+        1);
+  }
+};
+
+SqueezeNetImpl::SqueezeNetImpl(double version, int64_t num_classes)
+    : num_classes(num_classes) {
+  auto double_compare = [](double a, double b) {
+    return double(std::abs(a - b)) < std::numeric_limits<double>::epsilon();
+  };
+
+  if (double_compare(version, 1.0)) {
+    // clang-format off
+		features = torch::nn::Sequential(
```

Comment:
Can you de-dent these?

Reply:
These are left from my custom style. I'll remove all of these